lib/data_cycle/schedule_course_updates.rb

```diff
@@ -40,6 +40,12 @@ def enqueue_course_updates
 
     courses_to_update.each do |course|
       CourseDataUpdateWorker.update_course(course_id: course.id, queue: queue_for(course))
+
+      # if course isn't updated before, add first update flags
+      next if course.flags[:first_update]
```

Comment:
Sorry, I didn't notice this before... I think this should also skip setting the :first_update flag the course already has update logs from previous updates. Otherwise, existing courses in production will have inaccurate "first update" timestamps that aren't actually from the when they were first updated.